acorn-es7-plugin.js

```diff
@@ -72,6 +76,9 @@ function asyncAwaitPlugin (parser,options){
 					this.next() ;
 					var r = this.parseStatement(declaration, topLevel) ;
 					r.async = true ;
+					// TODO: This is probably required, but I don't understand nodent well enough to write a test case
+					// r.start = start;
+					// r.loc && (r.loc.start = startLoc);
```

Comment:
I am guessing these lines should be uncommented. It is for your `NON-STANDARD EXTENSION` for `options.asyncExits` for enodent.

Tell me how you want me to handle that.